power_grid_model_c/power_grid_model/include/power_grid_model/auxiliary/serialization/serializer.hpp

```diff
@@ -19,6 +19,20 @@
 #include <span>
 #include <string_view>
 
+constexpr void pack_inf(auto& packer_inf, auto& attr) {
+    using namespace msgpack;
+    using namespace std::string_view_literals;
+    using namespace power_grid_model::meta_data;
```

Comment:
not sure but i think both of these are now redundant

```suggestion
    using namespace std::string_view_literals;
```